provider/datagen/tests/data/baked/macros/percent_essentials_v1.rs.data

```diff
@@ -9,15 +9,15 @@ macro_rules! __impl_percent_essentials_v1 {
         #[clippy::msrv = "1.67"]
         const _: () = <$provider>::MUST_USE_MAKE_PROVIDER_MACRO;
         #[clippy::msrv = "1.67"]
-        impl icu_provider::DataProvider<icu::experimental::dimension::provider::percent::PercentEssentialsV1Marker> for $provider {
-            fn load(&self, req: icu_provider::DataRequest) -> Result<icu_provider::DataResponse<icu::experimental::dimension::provider::percent::PercentEssentialsV1Marker>, icu_provider::DataError> {
-                static UND: <icu::experimental::dimension::provider::percent::PercentEssentialsV1Marker as icu_provider::DataMarker>::Yokeable = icu::experimental::dimension::provider::percent::PercentEssentialsV1 { number_index: 0u8, percent_sign_affixes: icu::experimental::dimension::provider::percent::PercentAffixesV1 { prefix: alloc::borrow::Cow::Borrowed(""), suffix: alloc::borrow::Cow::Borrowed("") }, percent_sign_symbol: alloc::borrow::Cow::Borrowed("%"), percent_symbol_index: 5u8, standard: alloc::borrow::Cow::Borrowed("#,##0%") };
-                static VALUES: [&<icu::experimental::dimension::provider::percent::PercentEssentialsV1Marker as icu_provider::DataMarker>::Yokeable; 1usize] = [&UND];
+        impl icu_provider::DataProvider<icu::experimental::dimension::currency::provider::percent::PercentEssentialsV1Marker> for $provider {
```

Comment:
@robertbastian : In this file, I have to make the changes manually, even though, these changes should came from databake ? do you have any thoughts about this ?

Reply:
this is generated with `cargo make testdata`